stylegan_code_finder/visualization/segmentation_plotter_lightning.py

```diff
@@ -0,0 +1,68 @@
+import json
+from pathlib import Path
+from typing import List
+import os
+import numpy
+import torch
+from PIL import ImageColor
+from PIL import Image
+import torchvision
+from pytorch_training.images.utils import make_image
+from utils.data_loading import fill_plot_images
+
+
+class SegmentationPlotter:
+    def __init__(self, plot_data_loader, config):

```

Comment:
I would make this plotter independent from some external data loader. Instead, use the normal validation dataloader, but call the plotter only for the first `n` batches/images